examples/gtfs_reader.rs

```diff
@@ -20,7 +20,7 @@ extern crate navitia_model;
 extern crate serde_json;
 
 fn main() {
-    let objects = navitia_model::gtfs::read(".");
+    let objects = navitia_model::gtfs::read(".", None);
```

Comment:
you can do the same as `example/test.rs` with `run` function

And we should do the same in `bin/gtfs2ntfs.rs`